io.asgardeo.tomcat.oidc.sample/src/main/webapp/home.jsp

```diff
@@ -16,18 +16,29 @@
   ~ under the License.
   --%>
 
-<%@page import="io.asgardeo.java.oidc.sdk.SSOAgentConstants" %>
-<%@page import="io.asgardeo.java.oidc.sdk.bean.SessionContext" %>
+<%@ page import="io.asgardeo.java.oidc.sdk.SSOAgentConstants" %>
+<%@ page import="io.asgardeo.java.oidc.sdk.bean.SessionContext" %>
 <%@ page import="io.asgardeo.java.oidc.sdk.bean.User" %>
+<%@ page import="io.asgardeo.java.oidc.sdk.config.model.OIDCAgentConfig" %>
+<%@ page import="java.util.ArrayList" %>
 <%@ page import="java.util.HashMap" %>
 <%@ page import="java.util.Map" %>
+<%@ page import="net.minidev.json.JSONObject" %>
+<%@ page import="com.nimbusds.jwt.SignedJWT" %>
 
 <%
     final HttpSession currentSession = request.getSession(false);
     final SessionContext sessionContext = (SessionContext)
             currentSession.getAttribute(SSOAgentConstants.SESSION_CONTEXT);
     final String idToken = sessionContext.getIdToken();
-    
+
+    OIDCAgentConfig configfile = (OIDCAgentConfig) getServletContext().getAttribute(SSOAgentConstants.CONFIG_BEAN_NAME);
+    String scopes = configfile.getScope().toString();
```

Comment:
How would this work when there are multiple scopes? Does it return a comma-separated string?
A better approach would be to keep a String[] list and get a string list from the scope object.
`scopes.toStringList()`

Reply:
It returns a comma separated string. It is handled in  <script>